integreat_cms/cms/templates/analytics/analytics_overview.html

```diff
@@ -7,19 +7,18 @@
         <h1 class="heading">{% trans 'Analytics' %}</h1>
         <p class="py-3 text-lg"> {% trans 'Here you will find a set of tools to check the success and correctness of your app. Click on the Details button of a tool to get further information.' %}</p>
         <div class="xl:flex gap-3 pt-4">
-            <div class="xl:w-1/2">
-                <!-- Broken Link Widget -->
-                {% include "analytics/_broken_links_widget.html" %}
-                <!-- Translation Status Widget -->
-                {% include "analytics/_translation_overview_widget.html" %}
```

Comment:
Maybe we still need the dummy widget for the translation coverage, because it looks a bit weird to have an overview site for analytics, but then there is only the broken link checker aligned to the right and the translation coverage is only visible when clicking on the menu item? the current solution is more confusing than a dummy widget for the translation coverage I think...

Reply:
You are right, it's back again.